magefiles/magefile.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 var minGoVersion = "1.20"
-var tinygoMinorVersion = "0.29"
+var tinygoMinorVersion = "0.30"
```

Comment:
BTW I have been thinking compatibility has been good for the past 3-4 releases I think. We could probably make this check a minimum similar to the Go version check.

Reply:
sounds like something we could do, any advice @deadprogram?

Reply:
I opened https://github.com/corazawaf/coraza-proxy-wasm/pull/232 if we feel we can relax tinygo version check